pytorch_toolkit/object_detection/tests/nncf_tests_face_detection.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import os
+
+from common.test_case import create_nncf_test_case
+
+
+class FaceDetection0200NNCFnocompression(
+        create_nncf_test_case(
+            'face-detection',
+            'face-detection-0200',
+            os.path.dirname(__file__) + '/../../../data/airport/annotation_faces_train.json',
```

Comment:
I think the proper way is to define a variable on top of the file, like TEST_DATA_PATH=os.path.dirname(__file__) + '/../../../data/airport', and use it everywhere in the file, e.g. TEST_DATA_PATH + 'annotation_faces_train.json'

Reply:
I copied it from Ilya's tests. I'd prefer to use the same approach in all tests, and at the moment I'd prefer do not change this approach (since it works)